pages/linux/ss.md

```diff
@@ -2,26 +2,26 @@
 
 > Utility to investigate sockets.
 
-- Dump all TCP/UDP/RAW/UNIX sockets:
+- Show all TCP/UDP/RAW/UNIX sockets:
 
 `ss -a {{-t/-u/-w/-x}}`
```

Comment:
I don't think we should outright ban this construction; in cases like this where the commands are identical except for the option, it seems justifiable. But we need to agree that it's acceptable and when, so we don't impose restrictions on others that we don't follow universally.